lib/screens/MainScreen.dart

```diff
@@ -20,96 +20,226 @@ class SizeConfig {
+                          '50%',
+                          style: TEXT_THEME.headline1,
                         ),
                         Text(
-                            '1100/2200ml',
-                            style: TEXT_THEME.headline2,
+                          '1100/2200ml',
+                          style: TEXT_THEME.headline2,
                         ),
                       ],
-                )
-                ),
+                    )),
               ),
               Padding(
-                padding: EdgeInsets.only(top: SizeConfig.blockSizeVertical*3.52),
-                child:  PanelButton(
-                    width: SizeConfig.blockSizeHorizontal*90.5,
-                    height: SizeConfig.blockSizeVertical*13.17,
-                    child: SimpleButton(
-                      width: SizeConfig.blockSizeHorizontal*19.5,
-                      height: SizeConfig.blockSizeVertical*10.8,
-                    )
-                )
+                padding:
+                    EdgeInsets.only(top: SizeConfig.blockSizeVertical * 3.52),
+                child: PanelButton(
+                  width: SizeConfig.blockSizeHorizontal * 90.5,
+                  height: SizeConfig.blockSizeVertical * 13.17,
+                  child: ListView(
+                    scrollDirection: Axis.horizontal,
+                    children: <Widget>[
+                      SizedBox(width: SizeConfig.blockSizeVertical * 1.6),
+                      new SimpleButton(
+                        width: SizeConfig.blockSizeVertical * 10.8,
+                        height: SizeConfig.blockSizeVertical * 10.8,
+                        margin: EdgeInsets.symmetric(
+                            vertical: SizeConfig.blockSizeVertical * 1.6),
+                        child: Text(
+                          '1',
+                          style: TEXT_THEME.headline1,
+                        ),
+                      ),
+                      SizedBox(width: SizeConfig.blockSizeVertical * 1.6),
+                      new SimpleButton(
+                        width: SizeConfig.blockSizeVertical * 10.8,
+                        height: SizeConfig.blockSizeVertical * 10.8,
+                        margin: EdgeInsets.symmetric(
+                            vertical: SizeConfig.blockSizeVertical * 1.6),
+                        child: Text(
+                          '2',
+                          style: TEXT_THEME.headline1,
+                        ),
+                      ),
+                      SizedBox(width: SizeConfig.blockSizeVertical * 1.6),
+                      new SimpleButton(
+                        width: SizeConfig.blockSizeVertical * 10.8,
+                        height: SizeConfig.blockSizeVertical * 10.8,
+                        margin: EdgeInsets.symmetric(
+                            vertical: SizeConfig.blockSizeVertical * 1.6),
+                        child: Text(
+                          '3',
```

Comment:
Its code duplication. If you use same widgets with the same params you can extract them to separete class and reuse them.